src/types/AddAdmin.ts

```diff
@@ -1,8 +1,8 @@
 // This file was generated by [ts-rs](https://github.com/Aleph-Alpha/ts-rs). Do not edit this file manually.
-import type { PersonId } from "./PersonId";
+import type { LocalUserId } from "./LocalUserId";
 
 export interface AddAdmin {
-  person_id: PersonId;
+  local_user_id: LocalUserId;
```

Comment:
Is `LocalUserId` supposed to mean a user local to the instance, or the currently signed in user?

Reply:
User local to instance.